### ISSUE TITLE
Default to on when powered up

### ISSUE THREAD

**Issue by ghoggan** [arendst/Sonoff-MQTT-OTA#11]
It would be good to have the ability to set the device to switch itself to on when powered up

**Comment by arendst** [comment]
It does when it was on before power down.

**Comment by ghoggan** [comment]
Hi Theo
What I had in mind was to power on at startup regardless of the previous state.
This would allow the unit to be easily installed inline with existing lighting circuit wiring.
Normal control would be via Mqtt or Alexa, fallback if automation fails would be to use the existing light switch. Helps the WAF (wife acceptance factor)
I realise there is allowance for external switch input but this would leave the existing wiring more or less intact.

Thanks for the response,
Glenn

**Comment by ghoggan** [comment]
Hi Theo
Meant to post to https://github.com/arendst/Sonoff-MQTT-OTA-Arduino

Glenn

**Comment by 8666** [comment]
Yes than would be nice but there should be 3 contacts on the output. Now
only normaly-open and com are available

**Comment by malhal** [comment]
Default to opposite is maybe what you want. And a push switch that breaks circuit.
